From 6572e098d5463bdcdcb9563ee9a736efed4a01c9 Mon Sep 17 00:00:00 2001
From: Justin Bronder <jsbronder@gmail.com>
Date: Tue, 24 Nov 2009 10:27:51 -0500
Subject: [PATCH] Fix pkgconfig for mpich2-ch3 v1.2.1

Variables were undefined, see http://bugs.gentoo.org/257821

$ pkg-config --list-all
[lots of output, then]
Variable 'exec_prefix' not defined in '/usr/lib/pkgconfig/mpich2-ch3.pc'
---
 src/packaging/pkgconfig/mpich2-ch3.pc.in |    8 ++++++--
 1 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/packaging/pkgconfig/mpich2-ch3.pc.in b/src/packaging/pkgconfig/mpich2-ch3.pc.in
index b061464..275b535 100644
--- a/src/packaging/pkgconfig/mpich2-ch3.pc.in
+++ b/src/packaging/pkgconfig/mpich2-ch3.pc.in
@@ -1,8 +1,12 @@
 # this gives access to the mpich2 header files
+prefix=@prefix@
+exec_prefix=@exec_prefix@ 
+libdir=@libdir@
+includedir=@includedir@
 
 Name: mpich2
 Description: MPICH2 is an implementation of the Message-Passing Interface (MPI). The goals of MPICH2 are to provide an MPI implementation for important platforms, including clusters, SMPs, and massively parallel processors. It also provides a vehicle for MPI implementation research and for developing new and better parallel programming environments.
 Version: @MPICH2_VERSION@
 Requires: 
-Libs: -L@libdir@ -lmpich -lmpichcxx -lopa @LIBS@
-Cflags: @CFLAGS@ -I@includedir@ -DMPICH_IGNORE_CXX_SEEK
+Libs: -L${libdir} -lmpich -lmpichcxx -lopa @LIBS@
+Cflags: @CFLAGS@ -I${includedir} -DMPICH_IGNORE_CXX_SEEK
-- 
1.6.3.3

